.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Docs to Github Pages
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Master
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js for use with actions
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Build docs
+        run: |
+          npm run build-docs
+          npm run publish-docs
```

Comment:
Uma alternativa, é utilizar o próprio github para fazer o deploy no gh-pages. https://github.com/marketplace/actions/deploy-to-github-pages

Reply:
Criado uma issue p isso!